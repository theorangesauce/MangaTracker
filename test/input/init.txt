Series 1
1
Author 1
Publisher 1
Alternate Name 1
N
Series 2
1-128
Author 1
Publisher 2
Alternate Title 2
Y
Series 3
1-3, 5-7, 9-12
Author 2
Publisher 2
Alternate Title 3
N
Series 4
2-24
Author 3
Publisher 1
Alternate Title 4
N
Series 5
None
Author 2
Publisher 1
Alternate Title 5
N
Last Series
1-4
Author 3
Publisher 2
Alternate Name 6
Y

L
A
X
--END INIT--
