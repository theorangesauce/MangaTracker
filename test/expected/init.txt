Default List:
  Last Series
  Series 1
  Series 2
  Series 3
  Series 4
  
